info.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "fortiNDRCloud",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.2.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "FortiNDR Cloud",
+    "dependencies": [
+        {
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": "2.1.0"
+        },
+        {
+            "name": "socSimulator",
+            "type": "solutionpack",
+            "label": "SOC Simulator",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "This FortiSOAR Solution Pack for FortiNDR Cloud offering helps customers to readily orchestrate the investigation of detection.",
+    "help": null,
```

Comment:
Please add the doc link: https://github.com/fortinet-fortisoar/solution-pack-fortindr-cloud/blob/release/1.0.0/README.md